server_addon/maya/client/ayon_maya/plugins/publish/extract_ornatrix_cache.py

```diff
@@ -0,0 +1,88 @@
+import os
+import json
+from ayon_maya.api import lib
+from ayon_maya.api import plugin
+from maya import cmds
+
+
+class ExtractOxCache(plugin.MayaExtractorPlugin):
+    """Producing Ornatrix cache files using scene time range.
+
+    This will extract Ornatrix cache file sequence and fur settings.
+    """
+
+    label = "Extract Ornatrix Cache"
+    families = ["oxrig", "oxcache"]
+
+    def process(self, instance):
+        cmds.loadPlugin("Ornatrix.mll", quiet=True)
+        # Define extract output file path
+        ox_nodes = cmds.ls(instance[:], long=True)
+        dirname = self.staging_dir(instance)
+        attr_values = instance.data["creator_attributes"]
+        # Start writing the files for snap shot
+        # <NAME> will be replace by the Yeti node name
```

Comment:
This doesn't seem to be the case for ornatrix and comment is copied from yeti logic?
```suggestion
```